recipes/icclim/meta.yaml

```
@@ -0,0 +1,60 @@
+{% set name = "icclim" %}
+{% set version = "5.0.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/icclim-{{ version }}.tar.gz
+  sha256: 653cdad97fe565db3b71bab74a797ba394074e2838bd0d5343050330de92d09b
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
+    - setuptools
+    - xarray
+    - xclim >=0.33
+  run:
+    - python >=3.8
+    - cftime
+    - dask
+    - netCDF4
+    - numpy
+    - pyyaml
+    - xarray
+    - xclim >=0.33
+
+test:
+  requires:
+    - pip
+    - pytest
+  imports:
+    - icclim
+    - icclim.models
+  commands:
+    - pip check
```

```suggestion
test:
  requires:
    - pip
  imports:
    - icclim
    - icclim.models
  commands:
    - pip check
```
Pytest is unused, right? Your tests are only imports and dependency checking.